plugins/SlideExplorer/src/main/java/org/micromanager/slideexplorer/Hub.java

```diff
@@ -225,7 +215,6 @@ public void zoomBy(int zoomStep /* +-1 */, Point zoomTargetOffScreen) {
    }
 
    public void zoomTo(int zoomLevel, Point zoomTargetOffScreen) {
-      int originalZoomLevel = zoomLevel_;
       zoomLevel_ = zoomLevel;
```

Comment:
```suggestion
      int originalZoomLevel = zoomLevel_;
      zoomLevel_ = zoomLevel;
```

This would have introduced a bug.